Automattic/wp-calypso
# Add About Page and Contact Page guided tours

This PR adds guided tours triggered from the onboarding checklist.

# Testing

* Visit `/checklist/<domain>` for your site
* Click 'Do It' for the about page task (whether completed or not - the tour should show)
* Verify that the following steps appear in the tour:

![about-page](https://user-images.githubusercontent.com/1926/34092433-26737208-e40f-11e7-857a-54a82febdcb0.gif)

* Click 'Do It' for the about contact task (whether completed or not - the tour should show)
* Verify that the same steps appear in the tour.

# Issues

* first click on the checklist item does not launch the tour
